ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopicBackupImpl.java

```diff
@@ -209,7 +215,7 @@ private BackupReplayFile newReplayFile() {
 
     File latestBakFile = null;
     if (files != null) {
-      long latestTs = 0;
+      long latestTs = -1;
```

Comment:
is `latestTs` used anywhere?

Reply:
Changed. It was a change I did in the previous commit when changing the unit tests. It's not needed anymore.